tee-worker/litentry/core/vc-task/receiver/src/lib.rs

```diff
@@ -90,7 +90,8 @@ pub fn run_vc_handler_runner<ShieldingKeyRepository, A, S, H, O, Z, N>(
 	N::MetadataType: NodeMetadataTrait,
 {
 	let vc_task_receiver = init_vc_task_sender_storage();
-	let pool = ThreadPool::new().unwrap();
+	let n_workers = 12;
```

Comment:
I suggest putting this at the beginning with `const`.

Reply:
Can we make it parametrized instead of hardcoded ? what do you think ?